index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header id="header">
+            <nav class="navbar">
+                <h2>My Logo</h2>
```

Comment:
This is optional. It is based on your opinion. It is not part of the requirement.

Reply:
@annaeliana7 its part of the requirement, please wrap your logo with an `a `tag. because when we click on the logo it should redirect us to the home page.

Reply:
@sajid-munawar, can you please direct me to that section which mentioned the wrapping logo, because as I may know, biggest websites introduced that, And for me personally I don't feel like I have to use it because I have another approach. Thank you